src/main/java/edu/harvard/iq/dataverse/AuxiliaryFile.java

```diff
@@ -115,6 +142,25 @@ public String getChecksum() {
     public void setChecksum(String checksum) {
         this.checksum = checksum;
     }
-    
-    
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    /**
+     * @return "Differentially Private Statistics" (or the equivalent in French,
+     * etc.) rather than "DP" or null if this.type is null.
+     */
+    public String getTypeFriendly() {
+        if (this.type == null) {
+            return null;
```

Comment:
should null type, return other? (see the other note for a more generally suggested change, anyway)

Reply:
This was rewritten as part of 9e5f39089. Unspecified (null) type gets grouped as "other" in the UI.